index.mjs

```diff
@@ -106,7 +106,56 @@ import {hookAll, acquire, Hook} from 'fluture-hooks';
 // hasProp :: Object ~> String -> Boolean
 const hasProp = Object.prototype.hasOwnProperty;
 
-// callBootstrappers :: (Array (Bootstrapper), StrMap Resource) -> Hook (StrMap Resource)
+const error = message => new TypeError(message);
+const ok = Hook.of(null);
```

Comment:
This two functions are not used anymore.

Reply:
You are quite right.